libsplinter/src/admin/service/mailbox.rs

```diff
@@ -26,8 +26,8 @@ use super::messages::AdminServiceEvent;
 /// A simple entry for AdminServiceEvent values, marked with a timestamp
 #[derive(Debug, Eq, PartialEq, Clone)]
 pub struct EventEntry {
```

Comment:
This is an internal implementation detail of the memory store. It should actually be moved there.

Reply:
Moved this struct into the `memory` module